commerce_coordinator/settings/utils.py

```diff
@@ -39,8 +40,7 @@ def get_logger_config(logging_env="no_env",
         'disable_existing_loggers': False,
         'formatters': {
             'standard': {
-                'format': '%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] '
-                          '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] - %(message)s',
+                'format': format_string,
```

Comment:
For local logging, this change will result in an empty format string. Previously, it had a default format that was used for all the environments, but with this change, I am not sure what format the local setup will follow.

Reply:
We can set the default format in private.py that's why I am getting it in settings/local.py using `environ`

Reply:
We set the previously used formate as a default for local setup.

Reply:
the default would work for tests as well